parm/soca/berror/soca_ensrecenter.yaml

```diff
@@ -26,6 +29,57 @@ vertical geometry:
 
 add recentering increment: true
 recentering around deterministic: true
+sea ice recenter: true
+sea ice analysis:
+  pattern: '%mem%'
+  date: '{{ MARINE_WINDOW_END_ISO }}'
+  ocn_filename: "ocean.%mem%.nc"
+  ice_filename: "ice.%mem%.nc"
+  read_from_file: 1
+  basename: {{ ENSPERT_RELPATH }}/ens/
+  state variables:
+  - sea_water_potential_temperature
+  - sea_water_salinity
+  - eastward_sea_water_velocity
+  - northward_sea_water_velocity
+  - sea_water_cell_thickness
+  - sea_water_depth
+  - ocean_mixed_layer_thickness
+  - sea_ice_area_fraction
+  - sea_ice_thickness
+  - sea_ice_snow_thickness
+    
+sea ice variable change:
+  variable change name: Soca2Cice
+  arctic:
+    seaice edge: 0.4
+    shuffle: true
+    rescale prior:
+      rescale: true
+      min hice: 0.5
+      min hsno: 0.1
+  antarctic:
+    seaice edge: 0.4
+    shuffle: true
+    rescale prior:
+      rescale: true
+      min hice: 0.5
+      min hsno: 0.1
```

Comment:
note: these parameters should be the same as in soca2cice used in the marineanlckpt; ideally we would reference the same yaml section in both places.